service/windns/windns.go

```diff
@@ -288,3 +288,23 @@ func removeSingleNrtpRule(nrptRule string) {
 		log.Errorf("ERROR removing the nrpt rules: %v", err)
 	}
 }
+
+func CleanUpNetworkAdapterProfile() {
+	script := fmt.Sprintf(`$key="Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\NetworkList\Signatures\Unmanaged\*"
+Get-ItemProperty -Path $key | where {$_.FirstNetwork -match "ziti.*"} | Remove-Item
+
+$key="Microsoft.PowerShell.Core\Registry::HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\NetworkList\Profiles\*"
+Get-ItemProperty -Path $key | where {$_.ProfileName -match "Ziti.*"} | Remove-Item`)
+	log.Debugf("Cleaning up the Network Adapter profiles with: %s", script)
```

Comment:
at this point they are "Cleaned" up (past tense). might make sense to put this as the first line of the function

Reply:
We call the run function after this log statement, so it will be cleaned up after this statement.